src/test/java/org/sagebionetworks/bridge/schema/UploadSchemaKeyTest.java

```diff
@@ -14,77 +14,77 @@ public void equalsVerifier() {
-        new UploadSchemaKey.Builder().withStudyId("test-study").withRevision(42).build();
+        new UploadSchemaKey.Builder().withAppId("test-app").withRevision(42).build();
     }
 
     @Test(expectedExceptions = IllegalStateException.class, expectedExceptionsMessageRegExp = ".*schemaId.*")
     public void emptySchemaId() {
-        new UploadSchemaKey.Builder().withStudyId("test-study").withSchemaId("").withRevision(42).build();
+        new UploadSchemaKey.Builder().withAppId("test-app").withSchemaId("").withRevision(42).build();
     }
 
     @Test(expectedExceptions = IllegalStateException.class, expectedExceptionsMessageRegExp = ".*schemaId.*")
     public void blankSchemaId() {
-        new UploadSchemaKey.Builder().withStudyId("test-study").withSchemaId("   ").withRevision(42).build();
+        new UploadSchemaKey.Builder().withAppId("test-app").withSchemaId("   ").withRevision(42).build();
     }
 
     @Test(expectedExceptions = IllegalStateException.class, expectedExceptionsMessageRegExp = ".*revision.*")
     public void noRev() {
-        new UploadSchemaKey.Builder().withStudyId("test-study").withSchemaId("test-schema").build();
+        new UploadSchemaKey.Builder().withAppId("test-app").withSchemaId("test-schema").build();
     }
 
     @Test(expectedExceptions = IllegalStateException.class, expectedExceptionsMessageRegExp = ".*revision.*")
     public void negativeRev() {
-        new UploadSchemaKey.Builder().withStudyId("test-study").withSchemaId("test-schema").withRevision(-1).build();
+        new UploadSchemaKey.Builder().withAppId("test-app").withSchemaId("test-schema").withRevision(-1).build();
     }
 
     @Test(expectedExceptions = IllegalStateException.class, expectedExceptionsMessageRegExp = ".*revision.*")
     public void zeroRev() {
-        new UploadSchemaKey.Builder().withStudyId("test-study").withSchemaId("test-schema").withRevision(0).build();
+        new UploadSchemaKey.Builder().withAppId("test-app").withSchemaId("test-schema").withRevision(0).build();
     }
 
     @Test
     public void happyCase() {
-        UploadSchemaKey schemaKey = new UploadSchemaKey.Builder().withStudyId("test-study").withSchemaId("test-schema")
+        UploadSchemaKey schemaKey = new UploadSchemaKey.Builder().withAppId("test-app").withSchemaId("test-schema")
                 .withRevision(42).build();
-        assertEquals(schemaKey.getStudyId(), "test-study");
+        assertEquals(schemaKey.getAppId(), "test-app");
         assertEquals(schemaKey.getSchemaId(), "test-schema");
         assertEquals(schemaKey.getRevision(), 42);
-        assertEquals(schemaKey.toString(), "test-study-test-schema-v42");
+        assertEquals(schemaKey.toString(), "test-app-test-schema-v42");
     }
 
     @Test
     public void jsonSerialize() throws Exception {
         // start with JSON
         String jsonText = "{\n" +
-                "   \"studyId\":\"test-study\",\n" +
+                "   \"appId\":\"test-app\",\n" +
```

Comment:
The important part is the JSON serialization/deserialization. We should add the JSON annotations so that we still use studyId instead of appId.

Reply:
I updated this to use @JsonAlias rather than another builder method (both work but the annotation is nicer).